ansible/roles/dropbear_initramfs/defaults/main.yml

```diff
@@ -36,6 +36,22 @@ dropbear_initramfs__base_packages:
 # configuration.
 dropbear_initramfs__packages: []
 
+                                                                   # ]]]
+# .. envvar:: dropbear_initramfs__config_path [[[
+#
+# Path to dropbear initramfs configuration files.
+dropbear_initramfs__config_path: '{{ "/etc/dropbear/initramfs"
+                                      if (ansible_distribution_release in ["bookworm"])
+                                      else "/etc/dropbear-initramfs" }}'
+
+                                                                   # ]]]
+# .. envvar:: dropbear_initramfs__config_file [[[
+#
+# Path to dropbear initramfs configuration file.
+dropbear_initramfs__config_file: '{{ dropbear_initramfs__config_path + "/dropbear.conf"
+                                      if (ansible_distribution_release in ["bookworm"])
+                                      else dropbear_initramfs__config_path + "/etc/dropbear-initramfs/config" }}'
```

Comment:
You should use parentheses to ensure the scope of the if else. When I test this on bullseye I get `/etc/dropbear-initramfs/etc/dropbear-initramfs` for some reason. Something like:

```yaml
dropbear_initramfs__config_file: '{{ dropbear_initramfs__config_path + "/" + (
                                     "config"
                                     if (ansible_distribution_release in ["list of all old releases"])
                                     else "dropbear.conf") }}'
```

Reply:
Thx for your code snippet. I have used your style within the commit.